content/docs/instrumenting/exposition_formats.md

```diff
@@ -113,13 +113,13 @@ format. The following conventions apply:
 See also the example below.
 
 ```
-# HELP api_http_request_count The total number of HTTP requests.
-# TYPE api_http_request_count counter
-http_request_count{method="post",code="200"} 1027 1395066363000
-http_request_count{method="post",code="400"}    3 1395066363000
+# HELP http_request_total The total number of HTTP requests.
+# TYPE http_request_total counter
+http_request_total{method="post",code="200"} 1027 1395066363000
+http_request_total{method="post",code="400"}    3 1395066363000
 
 # Escaping in label values:
-msdos_file_access_time_ms{path="C:\\DIR\\FILE.TXT",error="Cannot find file:\n\"FILE.TXT\""} 1.234e3
+msdos_file_access_time_seconds{path="C:\\DIR\\FILE.TXT",error="Cannot find file:\n\"FILE.TXT\""} 1.234
```

Comment:
Updated.

On Thu, Mar 17, 2016 at 7:01 PM, Brian Brazil <notifications@github.com>
wrote:

> In content/docs/instrumenting/exposition_formats.md
> <https://github.com/prometheus/docs/pull/350#discussion_r56592041>:
>
> >
> >  # Escaping in label values:
> > -msdos_file_access_time_ms{path="C:\\DIR\\FILE.TXT",error="Cannot find file:\n\"FILE.TXT\""} 1.234e3
> > +msdos_file_access_time_seconds{path="C:\\DIR\\FILE.TXT",error="Cannot find file:\n\"FILE.TXT\""} 1.234
>
> This should be a bigger number for a unix timestamp
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/prometheus/docs/pull/350/files/5f857a7af09d84127855e96672d4f6416bc6dca3#r56592041>
>